packages/components/ItemIcon/ItemIcon.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+const ItemIcon = ({ url }) =>
+  <img src={`https://www.google.com/s2/favicons?domain=${url}`} alt="" />;
+
+export default ItemIcon;
```

Comment:
let's use named imports and exports.
Default exports are deprecated and rewritten now

Reply:
Done